### ISSUE TITLE
[ProgressView] MDCProgressView does not have accessibility label and fails accessibility tests

### ISSUE THREAD

**Issue by material-mirror** [material-components/material-components-ios#6413]
This was filed as an internal issue. If you are a Googler, please visit [b/123068208](http://b/123068208) for more details.

---
#### Internal data
- Associated internal bug: [b/123068208](http://b/123068208)

**Comment by jverkoey** [comment]
Duplicate of https://github.com/material-components/material-components-ios/issues/7603
